src/components/linearProgressBar/LinearProgressBar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
this are not related to skeleton

Reply:
ok, but I don´t remember to commited that